src/Services/ContentService.php

```diff
@@ -1050,175 +1050,66 @@ public function getFiltered(
                 $filter->groupByField($groupBy);
             }
 
-            if (config('railcontent.use_elastic_search') == true) {
-                $filters = [];
-
-                if (!empty($includedUserStates)) {
-                    $includedContentsIdsByState = [];
-                    $includedContentsByState =
-                        $this->userContentProgressRepository->createQueryBuilder('up')
-                            ->where('up.user = :userId')
-                            ->andWhere('up.state IN (:state)')
-                            ->setParameter('userId', auth()->id())
-                            ->setParameter('state', $includedUserStates)
-                            ->getQuery()
-                            ->getResult();
-                    foreach ($includedContentsByState as $progress) {
-                        $includedContentsIdsByState[] =
-                            $progress->getContent()
-                                ->getId();
-                    }
-                }
-
-                $followedContents = [];
-                if ($getFollowedContentOnly) {
-                    $followedContents = $this->contentFollowsRepository->getFollowedContentIds();
-                    if (empty($followedContents)) {
-                        $resultsDB = new ContentFilterResultsEntity([
-                            'results' => $followedContents,
-                            'total_results' => 0,
-                            'filter_options' => [],
-                        ]);
-
-                        $results =
-                            CacheHelper::saveUserCache($hash, $resultsDB, Arr::pluck($resultsDB['results'], 'id'));
-
-                        return new ContentFilterResultsEntity($results);
-                    }
-                }
-
-                if (!empty($requiredUserStates)) {
-                    $requiredContentsByState = $this->userContentProgressRepository->getUserProgressForState(
-                        auth()->id(),
-                        $requiredUserStates[0]
-                    );
+            $resultsDB = new ContentFilterResultsEntity([
+                                                                'results' => $filter->retrieveFilter(),
+                                                                'total_results' => $pullPagination ?
+                                                                    $filter->countFilter() : 0,
+                                                                'filter_options' => $pullFilterFields ? $this->getFilterOptions($filter, $includedTypes) : [],
```

Comment:
I think these were unnecessarily indented 
```suggestion
                'results' => $filter->retrieveFilter(),
                'total_results' => $pullPagination ?
                    $filter->countFilter() : 0,
                'filter_options' => $pullFilterFields ? $this->getFilterOptions($filter, $includedTypes) : [],
```